File: 0-1-Factorial.cs
static int Factorial(int n)
{
    // Breaking Condition
    if (n < 0)
    {
        return -1;
    }
    else if (n == 0)
    {
        return 1;
    }
    else
    {
        return n * Factorial(n - 1);
    }
}

File: 0-1-Factorial.js
//console.log("Hello World!");

function factorial(value) {
    // Breaking condition
    if (value < 0) {
        return -1;
    } else if (value === 0){
        return 1;
    } else {
        return value * factorial(value - 1);
    }
}

console.log(factorial(4));